src/diff/children.js

```diff
@@ -113,9 +113,9 @@ export function diffChildren(
 
 		newDom = childInternal._dom;
 
-		if (childVNode.ref) {
+		if (childVNode.ref || oldVNodeRef) {
```

Comment:
Wondering if we even need the conditional here, or could just always assign.

Reply:
To me this felt like a performance bottleneck as we keep on piling into the array then for every element, now it's only for elements that once had a ref in the current or prev cycle